dashboard/run_dashboard_app.py

```diff
@@ -34,23 +35,20 @@
     html.Div(
         [
             html.H3("Maui"),
-            html.Div(
-                [
-                    html.H5("Current status"),
-                    html.Div(id="maui_node_usage"),
-                    html.H5("Current quota"),
-                    html.Div(id="maui_quota_usage"),
-                    html.H5("Current queue"),
-                    html.Div(id="maui_squeue_table"),
-                    html.H5("Daily core hour usage"),
-                    dcc.Graph(id="maui_daily_chours"),
-                    html.H5("Total core hour usage"),
-                    dcc.Graph(id="maui_total_chours"),
-                    dcc.Interval(id="interval_comp", interval=10 * 1000, n_intervals=0),
-                    html.H5("maui_daily_inodes"),
-                    dcc.Graph(id="maui_daily_inodes"),
-                ]
-            ),
+            html.Div(id="err"),
```

Comment:
This has its content set when display_err is called?

Reply:
yes, display_error(n) function output html_text to the html_div with id="err"